example/get_data.py

```diff
@@ -0,0 +1,71 @@
+import os
```

Comment:
not sure have a single `get_data` in `example/` is good idea. we may create a folder called `example/common` or `example/util`. there is an also simpler function to download cifar10 and mnist, e.g. https://github.com/dmlc/mxnet/blob/master/example/image-classification/train_cifar10.py#L9